frontend/src/css/ContentItem.styl

```diff
@@ -36,6 +36,8 @@
     align-items center
     &__icon
       font-size bigIconSize
+  &__actions
+    margin-inline-start 74px
```

Comment:
Can you add a comment to explain the magical number, and maybe add it to a const